homework/vector-of-shared-ptrs/vectorFunctions.cpp

```diff
@@ -0,0 +1,45 @@
+#include "vectorFunctions.hpp"
+#include <iostream>
+#include <vector>
+#include <memory>
+
+std::vector<std::shared_ptr<int>> generate(int count) {
+	std::vector<std::shared_ptr<int>> sharedvec;
+	for (int i = 0; i < count; i++) {
+		sharedvec.push_back(std::make_shared<int>(i));
+	}
+	return sharedvec;
+}
+
+void print(std::vector<std::shared_ptr<int>> vec) {
+
+	for (int i = 0; i < vec.size(); i++)
+	{
+		std::cout << *vec.at(i) << " ";
+	}
+	std::cout << "\n";
+}
+
+void add10(std::vector<std::shared_ptr<int>> vec) {
+	for (int i = 0; i < vec.size(); i++)
+	{
+        if(vec[i] != nullptr) {
```

Comment:
if (value) is enough. This will call implicit `bool` operator which compare this ptr with `nullptr`